server/scripts/main_script.py

```diff
@@ -0,0 +1,6 @@
+exec(open("py_add_users.py").read())
+exec(open("add_hunts.py").read())
+#for hunt_team remember to update the hunt_id in add_teams.py
+#according to the hunt_id in mongo comapss. 
+exec(open("add_teams.py").read())
```

Comment:
let's try to avoid using exec. This will technically work but is not best practice. Please wrap the other scripts in a function/class in their respective files and then import those functions into main_script.py and run them like that

Reply:
You don't necessarily have to wrap the other scripts in a function/class, just replace "exec(open("add_hunts.py").read())" with "import add_hunts"